components/automate-ui/src/app/pages/project/rules/project-rules.component.ts

```diff
@@ -78,48 +80,44 @@ export class ProjectRulesComponent implements OnInit, OnDestroy {
         this.store.select(updateStatus),
         this.store.select(getProjectStatus)
       ]).pipe(
-        takeUntil(this.isDestroyed),
-        map(([gStatus, uStatus, gpStatus]) => {
```

Comment:
❓I might be missing something, but the `pipe(x, map(y)).subscribe()` logic _seems_ to be equivalent to `pipe(x).subscribe(y)`. Isn't it?

There are a few more below 🔽